tests/interfaces/test_jacobian_products.py

```diff
@@ -67,7 +67,10 @@ def inner_execute_numpy(tapes):
     device_ps_jacs,
     device_native_jps,
     device_ps_native_jps,
-    lightning_vjps,
+    pytest.param(
+        lightning_vjps,
+        marks=pytest.mark.skip(reason="Lighting JPC needs to be updated for new API"),
+    ),
```

Comment:
TODO: Stop skipping after lightning supports VJP and `LightningVJP` is fully integrated with new device API